.github/workflows/push-master.yml

```diff
@@ -0,0 +1,31 @@
+name: any-pr
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push-master:
+    # Don't run on any commit that features [skip ci] in the commit message
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    name: Build, test and publish unstable release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.15.3]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm i
+    - run: npm run test
+    - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc 2> /dev/null
+        env:
+          NPM_TOKEN: ${{ secrets.NPMJS_TOKEN }}
+    - run: git config user.name "DIF CI"
+    - run: git config user.email "ci@identity.foundation"
+    - run: npm whoami
```

Comment:
Not needed

Reply:
Agree it is not strictly needed, however it gives a clean failure if login to the NPM registry fails, rather than kicking this to failing during publishing the package to npm, happy to remove though

Reply:
perhaps a comment to that effect would be wise to prevent continued questioning :)